dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -16,6 +16,11 @@ using namespace winrt::Windows::Globalization::DateTimeFormatting;
 
 namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
 {
+    bool AppNotificationBuilder::IsUrgentScenarioSupported()
+    {
+        return GetBuildNumber() >= 19041;
```

Comment:
Ugh. Version checks. That's a minefield (as in, a reliable answer is a minefield...)

Please use functions in dev\common\IsWindowsVersion.h e.g. `WindowsVersion::IsWindows10_19H10OrGreater()`. That has checks for >=19H1 and >=20H1, and if additional version checks are needed extend as appropos.

That avoids the minefield of GetVersion mechanisms which don't necessarily yield the expected answer